### ISSUE TITLE
child_info_fork::abort: address space needed by 'parameter.dll' (0x1000000) is already occupied

### ISSUE THREAD

**Issue by urig** [babun/babun#680]
Hi,

I've had babun running on my Windows 10 64-bit machine for quite some time. This week it started failing with the following error appearing as soon as I launch it:

> 0 [main] zsh 5404 child_info_fork::abort: address space needed by 'parameter.dll' (0x1000000) is 
> already occupied
> compaudit:132: fork failed: resource temporarily unavailable
> 0 [main] zsh 3900 child_info_fork::abort: address space needed by 'parameter.dll' (0x1000000) is already occupied
> compinit:531: fork failed: resource temporarily unavailable

I've scanned the list of BLODA and I only came up with Windows Defender. Having disabled it the problem persists. 

How can I fix this?

**Comment by coopermaa** [comment]
I encountered the sample problem this week. My environment is Windows 7 64-bit. The error message is a little different:

> 0 [main] zsh 8536 child_info_fork::abort: address space needed by 'stat.dll' (0x420000) is already occupied
zsh: fork failed: resource temporarily unavailable

>   0 [main] zsh 8692 child_info_fork::abort: address space needed by 'stat.dll' (0x420000) is already occupied
VCS_INFO_realpath:7: fork failed: resource temporarily unavailable

**Comment by sm0king** [comment]
me too.
how can i fix this?

**Comment by bcawrse** [comment]
Ditto.. I've only had Babun for a couple days on Win 7. I uninstalled and re-installed babun, and this problem persists.

**Comment by carlitorendon** [comment]
Ditto. Same issue. Came out of the blue. Was working fine one moment and then not.

**Comment by carlitorendon** [comment]
An update: a reboot seems to have fixed it for now (no other action was taken in regards to babun). We'll see if it appears again....

**Comment by Suzal3579** [comment]
Opened babun **Works Fine**
(Closed and reopened again - A blink of an eye.....)
This wild error appears.

Any fix found??

**Comment by carlitorendon** [comment]
have you tried re-basing from file explorer?

![image](https://user-images.githubusercontent.com/14046845/27598746-053229de-5b2c-11e7-889d-20a4f1c3bbfc.png)

double click the rebase batch file

**Comment by Suzal3579** [comment]
@carlitorendon tried both **rebase** and **update** didn't work.

**Comment by carlitorendon** [comment]
Yeah I've had a couple of those instances where it gets into a real funk. In those cases, if I remember correctly, I had to reboot, then as soon as i come back, run the updater. Worst case you may have to re-install :/

**Comment by Suzal3579** [comment]
@carlitorendon  and sir , you just saved my terminal . A simple reboot fixed my issue. Thanks.

**Comment by carlitorendon** [comment]
Glad it helped :)

**Comment by FranciscoJS12** [comment]
I just had this problem and **update** (inside C:\Users\username\.babun) was the way to fix it for me.
